src/main/resources/application.yml

```diff
@@ -18,7 +18,12 @@ spring:
     password: ${bookit.database.password:}
     url: ${bookit.database.url:}
     driver-class-name: ${bookit.database.driver:}
+  jpa:
+#    generate-ddl: false
+    show-sql: true
+    hibernate:
+      ddl-auto: validate
```

Comment:
these settings might change once/if we let JPA manage the schema... for now still driven by schema.sql

Reply:
I kind of fear us getting into a "deadlocked" deployment again if we fail to sync schema.sql and annotations and thus startup fails.  You?

Reply:
agree... hoping the existence of schema.sql is short term... next go around will eliminate it